src/OctoshiftCLI.Tests/BbsApiTests.cs

```diff
@@ -0,0 +1,97 @@
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OctoshiftCLI.Extensions;
+using Xunit;
+
+namespace OctoshiftCLI.Tests
+{
+    public class BbsApiTests
+    {
+
+        private readonly Mock<OctoLogger> _mockOctoLogger = TestHelpers.CreateMock<OctoLogger>();
+        private readonly Mock<BbsClient> _mockBbsClient = TestHelpers.CreateMock<BbsClient>();
+
+        private readonly BbsApi sut;
+
+        private const string BBS_SERVICE_URL = "http://localhost:7990/rest/api/1.0";
+        private const string PROJECT_KEY = "TEST";
+        private const string SLUG = "test-repo";
+        private const long EXPORT_ID = 12345;
+
+        public BbsApiTests()
+        {
+            sut = new BbsApi(_mockBbsClient.Object, BBS_SERVICE_URL, _mockOctoLogger.Object);
+        }
+
+        [Fact]
+        public async Task StartExport_Should_Return_ExportId()
+        {
+            var endpoint = $"{BBS_SERVICE_URL}/migration/exports";
+            var requestPayload = new
+            {
+                repositoriesRequest = new
+                {
+                    includes = new[]
+                    {
+                        new
+                        {
+                            projectKey = PROJECT_KEY,
+                            slug = SLUG
+                        }
+                    }
+                }
+            };
+
+            var responsePayload = new
+            {
+                id = EXPORT_ID
+            };
+
+            _mockBbsClient.Setup(x => x.PostAsync(endpoint, It.Is<object>(y => y.ToJson() == requestPayload.ToJson())).Result).Returns(responsePayload.ToJson());
```

Comment:
I think you can change it to this which is the better way to do it: 
```suggestion
            _mockBbsClient.Setup(x => x.PostAsync(endpoint, It.Is<object>(y => y.ToJson() == requestPayload.ToJson()))).ReturnsAsync(responsePayload.ToJson());
```